frontend/components/ui/blocknote/editor/custom-blocknote.css

```
@@ -85,3 +85,121 @@
+.bn-container.light .bn-block-content[data-content-type="latex"] *,
+.bn-container.light
+  .bn-block[data-selected="true"]
+  .bn-block-content[data-content-type="latex"],
+.bn-container.light
+  .bn-block[data-selected="true"]
+  .bn-block-content[data-content-type="latex"]
+  * {
+  user-select: none !important;
+  -webkit-user-select: none !important;
+  background-color: transparent !important;
+  background: transparent !important;
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)